operations/installation.rst

```diff
@@ -0,0 +1,43 @@
+.. _installation-operation:
+
+============
+Installation
+============
+
+Omnivector provides a repository to install Slurm from, to make sure all charms
+install a supported Slurm version. The currently supported Slurm version is
+``20.11.8`` on CentOS 7 and ``20.11.7`` on Ubuntu Focal.
+
+Omnivector Slurm Distribution allows one to use a custom repository URL to
+install Slurm from. This is specially useful to allow using a local mirror, or
+cache, of packages.
+
+To specify a custom URL, set the value of ``custom-slurm-repo`` in your
+*bundle* file, for each charm, *before* deploying the system. For Ubuntu units,
+the URL must be a PPA valid URL. For CentOS 7 units, the URL can contain a
+``$basearch`` variable. For example:
+
+::
+
+   applications:
+     slurmd-centos:
+       charm: slurmd
+       num_units: 1
+       series: centos7
+       custom-slurm-repo: https://omnivector-solutions.github.io/repo/centos7/stable/$basearch
+     slurmd-ubuntu:
+       charm: slurmd
+       num_units: 1
+       series: focal
+       custom-slurm-repo: ppa:omnivector/osd-testing
+     ...
+
+
+This allows one to specify different repositories for testing new Slurm
+versions, or upgrading only specific charms.
+
+.. note::
+
+   The configuration ``custom-slurm- repo`` must be set *before* deploying the
```

Comment:
Extra space in ``custom-slurm-repo``?

Reply:
I wish my eyes were as good as yours, nice catch!